### DISCUSSION TITLE
Should Speak Touch announce the hint when there is text?

### DISCUSSION THREAD

**Discussion by Irineu333** [Irineu333/SpeakTouch#119] [Question]
Talkback always announces the hint. Unlike Jieshou, which only announces it when the text is empty.

**Comment by PatrykMis** [comment]
It definitely makes sense to announce the hint only if the edit text is empty. If the user has already entered input, we assume that they know what they are doing.

Similar behavior is found in screen readers like NVDA and JAWS for Windows, which only announce hint/placeholder text if the text field is empty.

**Reply by Irineu333** [reply]
I agree. So let's continue like this for now.
